string pot1
string button1
